thisweek/_posts/2022-03-07-changelog-119.adoc

```diff
@@ -0,0 +1,71 @@
+= Changelog #119
+:sectanchors:
+:page-layout: post
+
+Commit: commit:5fae65dd28b450a437ebc800a410164c3af1d516[] +
+Release: release:2022-03-07[]
+
+== Sponsors
+
+**Become a sponsor:** On https://opencollective.com/rust-analyzer/[OpenCollective] or
+https://github.com/sponsors/rust-analyzer[GitHub Sponsors].
+
+== A Message from the Team
```

Comment:
r? @matklad, @flodiebold, @Veykril

I'm not sure if you think it's fine to include such political statements, but not saying anything feels more wrong.